### PR TITLE
Fix push notifications

### PR DESCRIPTION
**Pull request by zacharyfmarion**
Fixed signing issue and improper use of regex so that opening a notification no longer crashes the app. We need to get this onto the app store ASAP.

### CODE REVIEW

File: iOS/chronreact/AppDelegate.m [hunk 1]
```diff
@@ -19,6 +19,8 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 {
   [BatchPush enableAutomaticDeeplinkHandling:NO];
   NSURL *jsCodeLocation;
+  
+  NSLog(@"%@", [BatchUser installationID]);
```
Comment (blakejkaplan) [inline iOS/chronreact/AppDelegate.m]: This is just a print statement right? Mind getting rid of it if you don't need it anymore?

File: iOS/chronreact/AppDelegate.m [hunk 2]
```diff
@@ -42,7 +44,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
    * generated by "Bundle React Native code and images" build step.
    */
 
-//   jsCodeLocation = [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
+//  jsCodeLocation = [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
```
Comment (blakejkaplan) [inline iOS/chronreact/AppDelegate.m:45]: If this is commented out we can get rid of it right?

Comment (zacharyfmarion) [reply iOS/chronreact/AppDelegate.m:45]: When you switch to a production build you switch between the two jsCodeLocation declarations so we want to keep this one

File: iOS/chronreact/AppDelegate.m (hunk 1 above)
Comment (zacharyfmarion) [reply iOS/chronreact/AppDelegate.m]: yeah I'll get rid of it

**Review by blakejkaplan — APPROVED** [review]

**Pull request merged**